airflow/providers/google/cloud/transfers/gdrive_to_gcs.py

```diff
@@ -74,8 +73,8 @@ class GoogleDriveToGCSOperator(BaseOperator):
     def __init__(
         self,
         *,
-        destination_bucket: str,
-        destination_object: str,
+        bucket_name: str,
+        object_name: str,
```

Comment:
This is a breaking change. While I agree that you suggestion is reasonable and I support it we will need to keep old arguments to preserve backward compatibly and we need to add deprecation warning.

Reply:
Okay